docs/examples/django/README.rst

```diff
@@ -0,0 +1,146 @@
+
+Once there run this command:
+
+`opentelemetry-python-autoinstrumentation python3 manage.py runserver`
+
+Execution of the client
+.......................
+
+Open up a new console and activate the previous virtual environment there too:
+
+`source django_auto_instrumentation/bin/activate`
+
+Go to `opentelemetry-python/ext/opentelemetry-ext-django/example`, once there
+run the client with:
+
+`python client.py hello`
+
+This should produce output similar to this one:
+
+.. code-block::
+
+    {
+        "name": "client-server",
+        "context": {
+            "trace_id": "0xa97f61b977a8cd4a21e3c43b8fae3708",
+            "span_id": "0x6290f0a6c49a5e9c",
+            "trace_state": "{}"
+        },
+        "kind": "SpanKind.INTERNAL",
+        "parent_id": "0x0af82d771221ac0a",
+        "start_time": "2020-04-26T01:48:51.793095Z",
+        "end_time": "2020-04-26T01:48:51.798319Z",
+        "status": {
+            "canonical_code": "OK"
+        },
+        "attributes": {},
+        "events": [],
+        "links": []
+    }
+    {
+        "name": "client",
+        "context": {
+            "trace_id": "0xa97f61b977a8cd4a21e3c43b8fae3708",
+            "span_id": "0x0af82d771221ac0a",
+            "trace_state": "{}"
+        },
+        "kind": "SpanKind.INTERNAL",
+        "parent_id": null,
+        "start_time": "2020-04-26T01:48:51.792938Z",
+        "end_time": "2020-04-26T01:48:51.798535Z",
+        "status": {
+            "canonical_code": "OK"
+        },
+        "attributes": {},
+        "events": [],
+        "links": []
+    }
+
+Go to the previous console, where the Django app is running. You should see
+output similar to this one:
+
+.. code-block::
+
+    {
+        "name": "home_page_view",
+        "context": {
+            "trace_id": "0xed88755c56d95d05a506f5f70e7849b9",
+            "span_id": "0x0a94c7a60e0650d5",
+            "trace_state": "{}"
+        },
+        "kind": "SpanKind.SERVER",
+        "parent_id": "0x3096ef92e621c22d",
+        "start_time": "2020-04-26T01:49:57.205833Z",
+        "end_time": "2020-04-26T01:49:57.206214Z",
+        "status": {
+            "canonical_code": "OK"
+        },
+        "attributes": {
+            "component": "http",
+            "http.method": "GET",
+            "http.server_name": "localhost",
+            "http.scheme": "http",
+            "host.port": 8000,
+            "http.host": "localhost:8000",
+            "http.url": "http://localhost:8000/?param=hello",
+            "net.peer.ip": "127.0.0.1",
+            "http.flavor": "1.1",
+            "http.status_text": "OK",
+            "http.status_code": 200
+        },
+        "events": [],
+        "links": []
+    }
+
+The last output shows spans automatically generated by the OpenTelemetry Django
+Instrumentation package.
+
+References
+----------
+
+* `OpenTelemetry Project <https://opentelemetry.io/>`_
```

Comment:
would be worth adding a link to django here

Reply:
Fixed